template/src/language/default-labels/videoCallScreenLabels.ts

```diff
@@ -0,0 +1,167 @@
+}
+
+export const VideoCallScreenLabels: I18nVideoCallScreenLabelsInterface = {
+  video: 'Video',
+  audio: 'Audio',
+  screenShareButton: 'Share',
+  record: 'Record',
+  recording: 'Recording',
+  switchCameraButton: 'Switch',
+  recordingNotificationLabel: 'Recording Started',
+  endCallButton: 'Hang Up',
+  participantsLabel: 'Participants',
+  groupChatLabel: 'Group',
+  privateChatLabel: 'Private',
+  chatMessageInputPlaceholder: 'Type your message..',
+  localScreenshareDefaultLabel: 'Your screenshare',
+  localUserDefaultLabel: 'You',
+  remoteUserDefaultLabel: 'User',
+  pstnUserLabel: 'PSTN User',
+  authenticationSuccessLabel: 'Authenticated Successfully!',
+  meetingCreatedNotificationLabel: 'Created',
+  joiningLoaderLabel: 'Starting Call. Just a second.',
+  oauthLoginLabel: 'Login using OAuth',
+  oauthProviderLabel: 'Please select an OAuth provider to login.',
+  copyMeetingInviteButton: 'Copy Meeting Invite',
+  pin: 'Pin',
+  language: 'Language',
+  hostControlsLabel: 'Host Controls',
+  muteAllAudioButton: 'Mute all audios',
+  muteAllVideoButton: 'Mute all videos',
+  screensharingActiveOverlayLabel: 'Your screen share is active.',
+  noOneJoinedLabel: 'No one has joined yet',
+  noLiveStreamingRequests: 'No streaming request(s)',
+  liveStreamingRequest: 'Streaming Request',
+  hostLabel: 'Host',
+  audienceLabel: 'Audience',
+  noUserFoundLabel: 'User not found',
+  raiseHandRequestMessage:
+    'You have raised your hand. Request sent to host for approval',
+  raiseHandRequestReceivedMessage: (name) => `${name} has raised their hand`,
+  raiseHandRequestAcceptedMessage:
+    'Your request was approved, unmute to start talking',
+  raiseHandRequestRejectedMessage: 'Your request was rejected by the host',
+  raiseHandRequestRecallMessage: (name) => `${name} has lowered their hand`,
+  raiseHandRequestRecallLocalMessage: 'You have lowered your hand',
+  raiseHandApprovedRequestRecallMessage:
+    'The host has revoked streaming permissions',
+  screenshareUserName: (name) => `${name}'s screenshare`,
+  recordingButton: (recording) => (recording ? 'Recording' : 'Record'),
+  raiseHandStatusText: (handStatus) =>
+    handStatus ? 'Lower hand' : 'Raise Hand',
+  messageSenderNotificationLabel: (name) => `From : ${name}`,
+  networkQualityLabel: (quality) => {
+    switch (quality) {
+      case 'unknown':
+        return 'Unknown';
+      case 'excellent':
+        return 'Excellent';
+      case 'good':
+        return 'Good';
+      case 'bad':
+        return 'Bad';
+      case 'veryBad':
+        return 'Very Bad';
+      case 'unpublished':
+        return 'Unpublished';
+      case 'loading':
+        return 'Loading';
+      default:
+        return 'Loading';
+    }
+  },
+  meetingInviteText: ({meetingName, id, url, pstn}) => {
+    let inviteContent = '';
+    if (url) {
+      if (url?.host) {
+        inviteContent += `Meeting - ${meetingName}\nURL for Attendee: ${url?.attendee}\nURL for Host: ${url?.host}`;
```

Comment:
break this down into chunks, this is getting messy and hard to read

Reply:
done